### PR TITLE
Split to components

### PR DESCRIPTION
**Pull request by aju**

### CODE REVIEW

File: components/simple-flip-clock/dist/js/main.min.js [hunk 1]
```diff
@@ -0,0 +1,132 @@
+/******/ (function(modules) { // webpackBootstrap
```
Comment (vergilius) [inline components/simple-flip-clock/dist/js/main.min.js]: not minified?

File: components/simple-flip-clock/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    maxNumber: maximal number that can be displayed - need when method `increment` or `decrement` will be used
+    minNumber: minimal number that can be displayed - need when method `increment` or `decrement` will be used
+    
+Methods
+        
+    update(number) - updates currently displayed number to provided one
+    decrement - decrements currently displayed number by 1
+    increment - increments currently displayed number by 1
+        
+[DEMO](https://brainly.github.io/simple-flip-clock/)
```
Comment (vergilius) [inline components/simple-flip-clock/README.md]: you still need to change repo name ♻️

File: components/simple-flip-clock/dist/js/main.min.js (hunk 1 above)
Comment (matzimowski) [inline components/simple-flip-clock/dist/js/main.min.js]: matify it.

**Comment by aju** [comment]
@matzimowski @vergilius updated

**Comment by vergilius** [comment]
👍

File: components/simple-flip-clock/README.md (hunk 2 above)
Comment (aju) [inline components/simple-flip-clock/README.md]: Will fix all not working urls in separate PR after changing repo name

File: README.md [hunk 3]
```diff
@@ -1,52 +1,7 @@
...
-    minNumber: minimal number that can be displayed - need when method `increment` or `decrement` will be used
-    
-Methods
-        
-    update(number) - updates currently displayed number to provided one
-    decrement - decrements currently displayed number by 1
-    increment - increments currently displayed number by 1
         
-[DEMO](https://brainly.github.io/simple-flip-clock/)
+[Simple flip clock](components/simple-flip-clock/README.md)
```
Comment (kdzwinel) [inline README.md]: I'd leave gifs in the main readme

**Comment by kdzwinel** [comment]
ok!

**Pull request merged**